database/repositories/helper.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
You don't need this file, you can use baseRepo. Refer to this https://github.com/nobejs/access/blob/master/src/repositories/tokens.js